packages/shiki-twoslash/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "devDependencies": {
     "@types/jest": "^25.1.3",
+    "fenceparser": "^1.1.0",
```

Comment:
I imagine this wants to be in "dependencies" instead?

Reply:
No no, shiki-twoslash only needs it for the type. Parsing is done only once in the remark package.